operators/netobserv-operator/1.0.1/manifests/netobserv-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole copy.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you please remve this file or substitute space in filename by `_`?

Reply:
oops sorry about that, that was a mistake. Should be removed now